bin/pygrb/pycbc_grb_inj_finder

```diff
@@ -84,67 +83,69 @@ def read_segment_files(segfiles):
         segmentlist()))
 
 
-def read_hdf5_triggers(inputfiles, verbose=False):
+def read_hdf5_triggers(inputfiles, trig_dict, verbose=False):
     """Merge several HDF5 files into a single file
 
     Parameters
     ----------
     inputfiles : `list` of `str`
         the paths of the input HDF5 files to merge
 
-    outputfile : `str`
-        the path of the output HDF5 file to write
+    trig_dict: dictionary 
+        dictionary with IFO or network identifier as keys
```

Comment:
It looks like `trig_dict` is no longer used: please remove here and from the function's arguments.